qa/L0_java_simple_example/test.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# Set up test files based on installation instructions
+# https://github.com/bytedeco/javacpp-presets/blob/master/tritonserver/README.md
+git clone https://github.com/bytedeco/javacpp-presets.git
+cd javacpp-presets
+mvn clean install --projects .,tritonserver
+mvn clean install -f platform --projects ../tritonserver/platform -Djavacpp.platform.host
+cd ..
+
+MODEL_REPO=`pwd`/../L0_simple_ensemble/models
+SAMPLES_REPO=`pwd`/javacpp-presets/tritonserver/samples
+BASE_COMMAND="mvn clean compile -f $SAMPLES_REPO exec:java -Djavacpp.platform=linux-x86_64"
+source ../common/util.sh
+
+rm -f *.log
+RET=0
+
+# Run with default settings
+$BASE_COMMAND -Dexec.args="-r $MODEL_REPO" >>client.log 2>&1
+if [ $? -ne 0 ]; then
+    RET=1
+fi
+
+if [ `grep -c "1 - 1 = 0" client.log` != "18" ]; then
```

Comment:
Are all input values for Input0 and Input1 = 1? Why 18 and not 16?

Reply:
In one of the cases that are run, they are. So that would be 16. In the other two inferences that run, input1 is always 1 and input0 has different values including one 1. So each of those have one "1 - 1 = 0". That gets you two 18.

Simple.java checks the outputs and fails if any are incorrect, so this is a check to make sure it ran until the end. "1 - 1 = 0" would be the last checked and printed value, so this requires simple.java to run and check the inference successfully.